src/physics/material/MaterialMd.hh

```diff
@@ -0,0 +1,26 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file MaterialMd.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include "base/OpaqueId.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Host-only metadata for a material.
+ */
+struct MaterialMd
+{
+    std::string name;    //!< Material name
+    std::string formula; //!< Chemical formula (optional)
```

Comment:
I haven't include the chemical formula in the geant-exporter. I'll get that done.

Reply:
Or I can delete this... it’s not super usrful

Reply:
Side note: apparently no one cares about chemical formulas. It is not mandatory in the G4Material class (not in the constructor, but setter and getter are in place), so I guess that's the reason. The cms geometry has defined the chemical formula for a grand total of zero materials.

Reply:
Ok I’ll delete it then. I only had it because @whokion mentioned it during our group collaboration day.

Reply:
@sethrj I mentioned the mater state, {solid, liquid, gaseous},  not chemical formula.